PokemonAPI/PokemonAPI/Controllers/AuthenticationController.cs

```diff
@@ -26,11 +27,19 @@ public AuthenticationController(IAuthenticationService authenticationService)
         [HttpGet("SignIn/{username}/{password}")]
         public async Task<IActionResult> SignIn(string username, string password, CancellationToken cancellationToken = default)
         {
-            var token = await _authenticationService.SignIn(username, password, cancellationToken);
+            try
+            {
+                var token = await _authenticationService.SignIn(username, password, cancellationToken);
 
-            Response.Cookies.Append(JwtBearerDefaults.AuthenticationScheme, token);
+                Response.Cookies.Append(JwtBearerDefaults.AuthenticationScheme, token);
 
-            return Accepted();
+                return Accepted();
+            }
+            catch
+            {
+                throw new UnauthorizedAccessException();
+            }
+            
```

Comment:
Revert these changes. This is just leftover from testing. Unauthorized exception is already thrown at service level. No need to do try catch blocks in our controllers